bots/random_bot.py

```diff
@@ -19,3 +19,5 @@ def play_move(self, board):
 		legal_moves = list(board.legal_moves)
 		random_move = random.choice(legal_moves)
 		board.push(random_move)
+	def accept_surrender(self):
```

Comment:
you can just use the super class method for this since it doesn't need to be abstract.